inductiva/fluids/fluid_block/fluid_block.py

```diff
@@ -62,15 +62,20 @@ def __init__(self,
         """Initializes the fluid block scenario.
 
         Args:
-            density: The density of the fluid in kg/m^3.
-            kinematic_viscosity: The kinematic viscosity of the fluid, in m^2/s.
+            density: The density of the fluid in kg/m^3. Valid
+                range is [400, 2000] Kg/m3.
+            kinematic_viscosity: The kinematic viscosity of the fluid, in m^2/s. 
+                Reference value for water is 1e-6 m^2/s.
             dimensions: The fluid block dimensions (in x, y, z), in meters.
             position: The position of the fluid block in the tank (in x, y, z),
               in meters.
             initial_velocity: The initial velocity of the fluid block (in x, y,
               z), in m/s.
         """
 
+        if not (400 <= density <= 2000):
```

Comment:
maybe you can do the assert instead    ` assert 400 <= density <= 1000 `

Reply:
assert can be deactivated and are more used for debugging, I believe. This needs to be performed at all run times to warn users there values are unfeasible!